tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/RbPath.java

```diff
@@ -192,12 +192,25 @@ public RbPath mapSegments(Function<? super String, String> fn) {
 
     // TODO: Remove this and isAlias() in favour of having properly typed paths.
     boolean isIntPath() {
-        String lastElement = segments.get(segments.size() - 1);
-        return lastElement.endsWith(":int") || lastElement.endsWith(":intvector");
+        return typeSuffixIsAnyOf(":int", ":intvector");
+    }
+
+    boolean isBinPath() {
```

Comment:
Improvement to support a "binary" type (for trie tables mainly).
This triggers different quoting and formatting in IcuData.